frontend/tests/jest/tests/sentry.test.tsx

```diff
@@ -33,20 +41,18 @@ describe('Sentry beforeSend', () => {
    });
 });
 
-const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
-function initSentry() {
+function initSentry(debug: boolean = false) {
    const mockBeforeSend = jest.fn();
 
    Sentry.init({
-      sampleRate: 1,
-      dsn: SENTRY_DSN,
+      debug,
+      dsn: 'https://00000000000000000000000000000000@o000000.ingest.sentry.io/0000000',
```

Comment:
should this be an env variable?

Reply:
I thought so too, but locally I did not have it set. Since this was the case undefined was pass in. Sentry (rightfully) just doesn't setup if it has nowhere to ping. FYI this value is from their own tests

Reply:
do jest tests not have access to env variables?

Reply:
They do, but the local dev default does not specify sentry being needed. This bypasses the need for checking the env in favor of hardcoding a value sentry will happily use and process errors under test regardless of a users env

Reply:
We don't specify NEXT_PUBLIC_SENTRY_DSN in `.env.test` because we don't want tests reporting to Sentry. We could achieve this a different way, but that was the intent